### PR TITLE
DM-28366: Added Docker files for building a base image for building a…

### PR DESCRIPTION
**Pull request by wvreeven**
…rm64 conda packages and for an arm64 deploy environment.

### CODE REVIEW

File: cycle/docker-compose.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: "3.7"
+version: "3.5"
```
Comment (tribeiro) [inline cycle/docker-compose.yaml:1]: Why you changed the version of the docker-compose file?

**Review by tribeiro — APPROVED** [review]
Made a few comments in line... take a look at them before merging.

File: build/develop-env/conda_package_builder_aarch64/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+ENV OSPL_HOME=/opt/OpenSpliceDDS/V${dds_community_version}/HDE/aarch64.linux
+
+RUN curl -sSL https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh -o miniforge.sh && \
+    chmod +x miniforge.sh && ./miniforge.sh -b -p ${WORKDIR}/miniforge && \
+    source /home/saluser/.setup_sal_env.sh && \
+    conda config --add channels conda-forge && \
+    conda config --add channels lsstts && \
+    conda install -y cython && \
+    conda install -y -c lsstts -c lsstts/label/dev python=3.8 && \
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply build/develop-env/conda_package_builder_aarch64/Dockerfile:53]: Modified it and removed the channels.

File: cycle/docker-compose.yaml (hunk 1 above)
Comment (wvreeven) [reply cycle/docker-compose.yaml:1]: That was a mistake. I changed it back.

**Pull request merged**